TombLib/TombLib/LevelData/Compilers/TombEngine/TombEngine.cs

```diff
@@ -262,7 +262,7 @@ private void WriteLevelTombEngine()
 
             using (var mediaStream = new MemoryStream())
             {
-                using (var writer = new BinaryWriterEx(mediaStream, true))
+                using (var writer = new BinaryWriter(mediaStream, System.Text.Encoding.ASCII, true))
```

Comment:
Why `System.Text.Encoding.ASCII` is used here? Old `BinaryWriterEx` was using `System.Text.Encoding.UTF8`.